### PR TITLE
feat(plugin-chart-echarts): implement cross filter in mixd-timeseries chart

### PR DESCRIPTION
**Pull request by stephenLYZ**
🏆 Enhancements
Add cross filter highlighting to MixdTimeSeries chart.

Both series support setting cross filter separately.
image

https://user-images.githubusercontent.com/11830681/127747100-889620ec-7ece-43f8-b9c1-6577a6e1ccf1.mov

### CODE REVIEW

File: plugins/plugin-chart-echarts/src/MixedTimeseries/EchartsMixedTimeseries.tsx [hunk 1]
```diff
@@ -16,10 +16,92 @@
...
+  setDataMask,
+  labelMap,
+  labelMapB,
+  groupby,
+  groupbyB,
+  selectedValues,
+  formData,
+  seriesBreakdown,
+}: EchartsMixedTimeseriesChartTransformedProps) {
+  const isFirstQuery = useCallback((seriesIndex: number) => seriesIndex < seriesBreakdown, [
```
Comment (stephenLYZ) [inline plugins/plugin-chart-echarts/src/MixedTimeseries/EchartsMixedTimeseries.tsx:37]: Here we need to determine which rawSeriesData was clicked (rawSeriesA or rawSeriesB)

**Review by villebro — APPROVED** [review]
LGTM and tested to work well! The only thing that's slightly confusing is that we have two different places where we need to enable x-filtering, but that's more due to the plugin implementation, not this feature. Some linting issues still need to be fixed to be able to remove the `@ts-ignore`s, but we can do that in a follow-up PR.

**Comment by zhaoyongjie** [comment]
Tested in my local. Looks good!

https://user-images.githubusercontent.com/2016594/127811048-a65195ca-b281-466c-b271-6c266fc98e5f.mov

**Review by zhaoyongjie — APPROVED** [review]
LGTM

**Pull request merged**